grammar.js

```diff
@@ -778,7 +778,10 @@ module.exports = grammar({
     ),
 
     // TODO: Include operators.
-    identifier: $ => /[a-zA-Z_]\w*/,
+    _plainid: $ => /[a-zA-Z_]\w*/,
+    // inside ` corresponds roughly to charNoBackQuoteOrNewline in Scala 3 EBNF
+    _char_no_back_quote_or_newline: $=> /[^\n`]+/,
+    identifier: $ => choice($._plainid, seq('`', $._char_no_back_quote_or_newline, '`')),
```

Comment:
If you use `seq(...)` wouldn't the current grammar allow newline in between the backquote and `$._char_no_back_quote_or_newline`? i.e.

```scala
val `

test-this` = 1
```

Reply:
You are right, I changed it from a regex, and it is indeed incorrect.
Using token.immediate also didn't work (and I didn't have the patience to experiment given that the grammar takes 2 minutes at least to regenerate)

It's back to original form in https://github.com/tree-sitter/tree-sitter-scala/pull/65/commits/0d94e6ee2a3ce4494e3edbf8ce70bac7e6e3ef71